### PR TITLE
DR dashboard - create new project

### PR DESCRIPTION
**Pull request by alex-cit**
related to #436 
Dashboard Redesign - screenshots update -  https://merginmaps.com/docs/manage/create-project/#create-a-project-through-merginmaps-com

### CODE REVIEW

File: src/manage/create-project/index.md [hunk 1]
```diff
@@ -80,52 +80,57 @@ Your new project should now be visible on the **Home** tab of the **Projects** s
 
 ## Create a project through merginmaps.com
```
Comment (MarcelGeo) [inline src/manage/create-project/index.md:81]: merginmaps.com -> dashboard ?

**Review by MarcelGeo — APPROVED** [review]

File: src/manage/create-project/index.md [hunk 2]
```diff
@@ -78,54 +78,59 @@ The project will be saved in the selected workspace.
 Your new project should now be visible on the **Home** tab of the **Projects** screen
 ![Mergin Maps mobile app home projects](../../tutorials/capturing-first-data/merginmaps-mobile-new-project-listed.jpg "Mergin Maps mobile app home projects")
 
-## Create a project through merginmaps.com
+## Create a project through Mergin Maps dashboard
```
Comment (raherin) [inline src/manage/create-project/index.md:81]: Edit text:
Create a project through the Mergin Maps dashboard

File: src/manage/create-project/index.md [hunk 3]
```diff
@@ -78,54 +78,59 @@ The project will be saved in the selected workspace.
...
-   - Once the files and folders are selected, press **Update** in the **Data Sync** window.
-   ![Upload files to Mergin Maps project](./web-project-upload.jpg "Upload files to Mergin Maps project")
+   Files can be uploaded using the <QGISPluginNameShort /> or by **Drag and drop**.
+   
+   If you are uploading files manually through the <DashboardShortLink />, do not forget to synchronise the changes by clicking **Update changes** in the **Data Sync**.
+   ![Upload files to Mergin Maps project](./dashboard-upload-new-project.jpg "Upload files to Mergin Maps project")
 
-### Clone an existing project through merginmaps.com
-You can also make a copy of projects you have access to or of a [public project](../permissions/#public-and-private-projects).
+### Clone an existing project through Mergin Maps dashboard
```
Comment (raherin) [inline src/manage/create-project/index.md:100]: Edit text:
Clone an existing project through the Mergin Maps dashboard

File: src/manage/create-project/index.md [hunk 4]
```diff
@@ -78,54 +78,59 @@ The project will be saved in the selected workspace.
...
+   ![Mergin Maps dashboard browse community projects](./dashboard-browse-public-projects.jpg "Mergin Maps dashboard browse community projects")
    
    If a project was shared with you via link, open the project link in your web browser.
    
-2. Using the search bar, find the project you would like to clone and click on it.
-   ![Find public Mergin Maps projects](./web-find-project.jpg "Find public Mergin Maps projects")
+2. You will see a list of public projects. Use the search bar to find a project you would like to clone and click on it.
+   ![Find public Mergin Maps projects](./dashboard-find-public-project.jpg "Find public Mergin Maps projects")
 
 3. The project's page opens. Click on the **Clone** button to clone the project.
```
Comment (raherin) [inline src/manage/create-project/index.md:129]: Edit text:
3. When the project's page opens, click on the **Clone** button to clone the project.

File: src/manage/create-project/index.md [hunk 5]
```diff
@@ -78,54 +78,59 @@ The project will be saved in the selected workspace.
...
-   ![Clone Mergin Maps project through web](./web-clone-project.jpg "Clone Mergin Maps project through web")
+   ![Clone public Mergin Maps project](./dashboard-clone-public-project.jpg "Clone public Mergin Maps project")
+   
+3. Now you can change the project's name and select the target workspace. Confirm by clicking **Clone project**.
```
Comment (raherin) [inline src/manage/create-project/index.md:132]: Should this be step 4?

**Review by raherin — CHANGES_REQUESTED** [review]

**Pull request merged**